Cargo.lock

```diff
@@ -10790,7 +10790,7 @@ dependencies = [
 
 [[package]]
 name = "vector"
-version = "0.44.0"
+version = "0.45.0"
```

Comment:
I'm not sure why this change has occurred, I didn't change any cargo files... Perhaps master is out of date??